### DISCUSSION TITLE
ADD - DeFooocus and mashb1t/Fooocus

### DISCUSSION THREAD

**Discussion by StabilityBro** [LykosAI/StabilityMatrix#599] [Ideas]
Nice Fork - https://github.com/ehristoforu/DeFooocus

**Comment by bsdesigner** [comment]
Well, then this version should be included in the build: https://github.com/mashb1t/Fooocus

https://www.youtube.com/watch?v=9qnl8SnjRGA

**Comment by aolko** [comment]
> Nice Fork - https://github.com/ehristoforu/DeFooocus

dead though
